tools/network_monitor.py

```
@@ -25,10 +25,14 @@ def network_monitor():
         print(f"Error: {e}")
 
+def check_platform_compatibility():
+    supported = True
+    warnings = []
 
-def main():
-    network_monitor()
-
+    try:
+        import psutil
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)